### PR TITLE
feat: Add title while linking the linear issue

### PR DESCRIPTION
**Pull request by muhsin-k**
**Preview**

CleanShot 2024-05-23 at 12 50 55@2x

### CODE REVIEW

**Review by scmmishra — DISMISSED** [review]

File: app/javascript/dashboard/components/widgets/conversation/linear/CreateOrLinkIssue.vue [hunk 1]
```diff
@@ -57,6 +63,20 @@ const { t } = useI18n();
 
 const selectedTabIndex = ref(0);
 
+const timeAgo = time => {
+  const unixTime = fromUnixTime(time);
+  return formatDistanceToNow(unixTime, { addSuffix: true });
+};
+const title = computed(() => {
```
Comment (scmmishra) [inline app/javascript/dashboard/components/widgets/conversation/linear/CreateOrLinkIssue.vue:70]: Won't this keep changing? Suppose I linked a month old conversation, and then take a look at it 3 months later. The title will still show "A month ago"

Comment (muhsin-k) [reply app/javascript/dashboard/components/widgets/conversation/linear/CreateOrLinkIssue.vue:70]: Good catch. We can update the preview link, but it requires some effort. For now, let's display only the conversation ID and contact name.

Comment (scmmishra) [reply app/javascript/dashboard/components/widgets/conversation/linear/CreateOrLinkIssue.vue:65]: Yes! Rest LGTM, let's get this PR merged 🔥

**Review by scmmishra — APPROVED** [review]

**Review by iamsivin — APPROVED** [review]
LGTM

**Pull request merged**